stats/opentelemetry/internal/tracing/custom_carrier_test.go

```diff
@@ -119,88 +159,67 @@ func (s) TestSet(t *testing.T) {
 			wantKeys:  []string{"key2", "key1"},
 		},
 		{
-			name:      "set non-grcp-trace-bin binary key",
+			name:      "set grpc-trace-bin binary key",
```

Comment:
The test name is Set..., so you can drop these set prefixes. Here and below.

Reply:
Done